test/test_torch.py

```diff
@@ -19396,7 +19414,12 @@ def generate_test_function(cls,
                            dtype_list,
                            dtype_cpu_list,
                            decorators) -> None:
-    def fn(self, device, dtype) -> None:
+    def fn_(self, device, dtype_, disable_tf32_on_fp32) -> None:
```

Comment:
Again, if you update the decorator, do you need to do any of this? Seems like the decorator can handle it?

Reply:
fixed